image-inspector.go

```diff
@@ -197,6 +242,29 @@ func getAuthConfigs(dockercfg, username, password_file string) *docker.AuthConfi
 	return imagePullAuths
 }
 
+func runOpenSCAP(inspectorMetadata *InspectorMetadata, inspectorOptions *ImageInspectorOptions) []byte {
+	log.Printf("OpenSCAP scanning %s", inspectorOptions.DstPath)
+	resultsDir, err := ioutil.TempDir(OSCAP_RESULT_DIR, "image-inspector-results-")
```

Comment:
@simon3z Is this a good solution for the exclusivity of the output file? AFAICS This function makes sure that no two image-inspectors will use the same resultsDir.